tgui/packages/tgui/interfaces/MODsuit.tsx

```diff
@@ -781,11 +781,11 @@ const ModuleSection = (props) => {
                     />
                   )}
                 </Table.Cell>
-                <Table.Cell textAlign="center">{module.idle_power}</Table.Cell>
+                <Table.Cell textAlign="center">{formatPower(module.idle_power)}</Table.Cell>
                 <Table.Cell textAlign="center">
-                  {module.active_power}
+                  {formatPower(module.active_power)}
                 </Table.Cell>
-                <Table.Cell textAlign="center">{module.use_power}</Table.Cell>
+                <Table.Cell textAlign="center">{formatEnergy(module.use_energy)}</Table.Cell>
```

Comment:
Right now this looks awful due to not enough space, so it wraps to a newline. I don't really know typescript, so I'm not sure what to do about this.

Reply:
Just increase the width of the whole window till is stops wrapping.

Use tgui-dev.bat in your bin directory to allow for dynamic changes to your UI mid game.

Reply:
Thanks. I'll try that. I assumed increasing the width wouldn't fix it, because extending the UI with my cursor didn't fix it due to the numbers portion of the UI having a fixed width.

Reply:
It didn't work, even when I applied extra width to the columns.